sagemaker-python-sdk/mxnet_horovod_mnist/code/config.json

```diff
@@ -0,0 +1 @@
+/home/ubuntu/amazon-sagemaker-examples/config.json
```

Comment:
How does this work? Paths for typical location is going to change btwn NI and Studio.

Reply:
I added a `REAEME.md` in `code/` to show people how to use those test scripts.